src/hyperscript_integration.jl

```diff
@@ -42,12 +42,16 @@ using .DOM
 
 # default turn attributes into strings
 attribute_render(session, parent, attribute, x) = string(x)
+attribute_render(session, parent, attribute, x::Nothing) = x
 function attribute_render(session, parent, attribute, obs::Observable)
     onjs(session, obs, js"""
     function (value){
         var node = $(parent);
         if(node){
-            if(node[$attribute] != value){
+            // we encode nothing == null to mean attribute not set!
+            if(value == null){
```

Comment:
This is a bit in contradiction with the way [HyperScript.jl](https://github.com/yurivish/Hyperscript.jl) works, AFAIU. From the last line in the docs:
> Using `nothing` as the value of a `DOM` attribute creates a valueless attribute, e.g. `<input checked />`.
If we have to add it "by hand" I wonder whether we are better off using `false` for this, or maybe a custom `Unset` type.

Alternatively, as WebIO allows to "unset" attributes with `nothing`, we could check with Hyperscript if they are open to changing that behavior.

Reply:
Ah! I guess I see what's going on here... I already messed with the attribute rendering, and made it ignore nothing (by rendering it to a string before giving it to hyperscript) :D

Reply:
I guess we need all those cases right?
* valueless attribute
* not set
* set to value
Maybe nothing == valueless, missing = removed?

Reply:
Good idea! The custom type is probably overkill. So actually the proper way to generate clean HTML would be to use `Observable{Union{Nothing, Missing}}` (AFAIU, valueless is the good HTML practice for boolean attributes).